lib/rules/property-no-unknown/__tests__/index.mjs

```diff
@@ -67,13 +67,21 @@ testRule({
 			endColumn: 7,
 		},
 		{
-			code: '.foo { *wdth: 100px; }',
+			code: '.foo { wdth: 100px; }',
 			message: messages.rejected('wdth'),
 			line: 1,
 			column: 8,
 			endLine: 1,
 			endColumn: 12,
 		},
+		{
+			code: '.foo { *wdth: 100px; }',
+			message: messages.rejected('wdth'),
+			line: 1,
+			column: 9,
+			endLine: 1,
+			endColumn: 13,
+		},
```

Comment:
The reported range for `*wdth` appears to have been incorrect before.

I've added an extra test with just `wdth` instead of the extra complexity of the leading `*`.